Command/NewEmailCommand.php

```diff
@@ -33,7 +35,7 @@ class NewEmailCommand extends ContainerAwareCommand
     protected function configure()
     {
         $this
-            ->setName('swiftmailer:email:send')
+            ->setName('swiftmailer:email:send') // BC with 2.7
```

Comment:
Can we use `self::$defaultName` here instead?

Reply:
nice catch. Yes we can :) doing it the lazy way right now.